### ISSUE TITLE
Unable to build from Git

### ISSUE THREAD

**Issue by clin1234** [crunchy-labs/crunchy-cli#417]
**Describe the bug**
Compiling from `master` yielded the following error"
```
error[E0603]: module `ssa` is private
  --> crunchy-cli-core\src\utils\download.rs:16:17
   |
16 | use rsubs_lib::{ssa, vtt};
   |                 ^^^ private module
   |
note: the module `ssa` is defined here
  --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\rsubs-lib-0.3.0\src\lib.rs:31:1
   |
31 | mod ssa;
   | ^^^^^^^

error[E0603]: module `vtt` is private
  --> crunchy-cli-core\src\utils\download.rs:16:22
   |
16 | use rsubs_lib::{ssa, vtt};
   |                      ^^^ private module
   |
note: the module `vtt` is defined here
  --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\rsubs-lib-0.3.0\src\lib.rs:33:1
   |
33 | mod vtt;
   | ^^^^^^^

For more information about this error, try `rustc --explain E0603`.
warning: `crunchy-cli-core` (lib) generated 1 warning
error: could not compile `crunchy-cli-core` (lib) due to 2 previous errors; 1 warning emitted
warning: build failed, waiting for other jobs to finish...
error: failed to compile `crunchy-cli v3.6.1 (https://github.com/crunchy-labs/crunchy-cli#53a710a3)`, intermediate artifacts can be found at `C:\Users\0BAC~1\AppData\Local\Temp\cargo-installQeedCn`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

**To Reproduce**
Steps / command to reproduce the behavior:
```
~> cargo install --git <repo URL>
```

**Expected behavior**

Successful compilation

**Client (please complete the following information):**
 - OS: Windows
 - Version: master (53a710a)

**Comment by Frooastside** [comment]
until the problem is resolved, you can try to change `rsubs-lib = ">=0.2.1"` to `rsubs-lib = "0.2.1"` in `crunchy-cli-core/Cargo.toml`

**Comment by bytedream** [comment]
Issue is now fixed
